src/framerate_gui.cpp

```diff
@@ -1079,3 +1096,15 @@ void ConPrintFramerate()
 		IConsolePrint(CC_ERROR, "No performance measurements have been taken yet.");
 	}
 }
+
+void ProcessPendingPerformanceMeasurements()
```

Comment:
would be good to have some comments explaining why only the sound performance stuff is ever "pending"

Reply:
I've added some explanatory text